site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -212,21 +196,13 @@ chrome.devtools.inspectedWindow.eval(
 );
 ```
 
-Alternatively, use the `useContentScriptContext: true` option for `inspectedWindow.eval()` to
-evaluate the expression in the same context as the content scripts. Calling `eval` with
-`useContentScriptContext: true` does not _create_ a content script context, so you must load a
+You can also use the `useContentScriptContext: true` option for `inspectedWindow.eval()` to
+evaluate the expression in the same context as the content scripts. To use this option, you must load a
```

Comment:
```suggestion
evaluate the expression in the same context as the content scripts. To use this option, use a [static content script declaration][static-declaration]
```

Reply:
The link is here. GitHub won't let me edit the line where this needs to go. Can you please add it.

https://developer.chrome.com/docs/extensions/mv3/content_scripts/#static-declarative

Reply:
Done.